packages/core-kit/src/nodes/run-javascript.ts

```diff
@@ -150,7 +152,7 @@ export function convertToNamedFunction({
 
 const DEFAULT_FUNCTION_NAME = "run";
 export const runJavascriptHandler = async (
-  { code, name, raw }: RunJavascriptInputs,
+  { code, name, raw, schema }: RunJavascriptInputs,
```

Comment:
```suggestion
  { code, name, raw }: RunJavascriptInputs,
```

(Don't think we need to use schema in invoke, only in describe)